spec/Store.spec.js

```diff
@@ -387,6 +414,70 @@ describe('Store', () => {
     })
   })
 
+  describe('createModel', () => {
+    it('creates a model obj with attributes', () => {
+      const todoData = {
+        attributes: { title: 'hello!' }
+      }
+      const todo = store.createModel('todos', 1, todoData)
+      expect(todo.id).toEqual(1)
+      expect(todo.title).toEqual('hello!')
+    })
+
+    it('creates a model obj with relatedToOne property', () => {
+      store.add('categories', { id: 5, name: 'Cat5' })
+      const todoData = {
+        attributes: { title: 'hello!' },
+        relationships: {
+          category: { data: { id: '5', type: 'categories' } }
+        }
+      }
+      const todo = store.createModel('todos', 1, todoData)
+      expect(todo.category.id).toEqual(5)
+      expect(todo.category.name).toEqual('Cat5')
+    })
+
+    it('creates a model with relatedToMany property', () => {
+      store.add('tags', { id: 3, label: 'Tag #3' })
```

Comment:
Maybe it would be better to set this to a variable and not use the explicit ids and instead do something like.
```JavaScript
const tag = store.add('tags', { label: 'Tag #3' })
const todoData = {
  attributes: { title: 'hello!' },
  relationships: {
    tags: { data: [{ id: tag.id, type: 'tags' }] }
  }
}
const todo = store.createModel('todos', 1, todoData)
expect(todo.id).toEqual(1)
expect(todo.tags[0].id).toEqual(tag.id)
expect(todo.tags[0].label).toEqual(tag.label)
```